.husky/pre-commit

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
+yarn prettier-check
```

Comment:
This is the only change for this MR, and the rest is just fixing the prettier errors by this command
```
yarn prettier-fix
```